policyengine_us/variables/gov/states/az/tax/income/exemptions/az_aged_exemption.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class az_aged_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona aged exemption"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        joint = filing_status == filing_status.possible_values.JOINT
+
+        p = parameters(period).gov.states.az.tax.income.exemptions.amount
+
+        age_head = tax_unit("age_head", period)
+        head_eligible = age_head >= p.min_age
```

Comment:
Then make `head_eligible = head_age_eligible & ~dependent_head` below
```suggestion
        head_age_eligible = age_head >= p.min_age
```